baseplate/frameworks/queue_consumer/__init__.py

```diff
@@ -0,0 +1,22 @@
+from baseplate.frameworks.queue_consumer._depr import BaseKombuConsumer
+from baseplate.frameworks.queue_consumer._depr import consume
+from baseplate.frameworks.queue_consumer._depr import Handler
+from baseplate.frameworks.queue_consumer._depr import KombuConsumer
+from baseplate.frameworks.queue_consumer._depr import WorkQueue
+from baseplate.frameworks.queue_consumer.kombu import KombuConsumerWorker as _ConsumerWorker
+from baseplate.lib import warn_deprecated
+
+warn_deprecated(
```

Comment:
does this not get called when importing the non-depr version?

Reply:
Gah, I was hoping it would only happen when you directly imported from `queue_consumer` but that doesn't appear to be the case.  😭